### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by nishasen**
Made changes in this branch, so opened this pull request again.

### CODE REVIEW

**Comment by nishasen** [comment]
image
This is the basic preview made some few changes in it, added user profile and logout tabs under user icon in the navbar, it may not be visible due to merging problem. Made the landing page responsive too

File: index.html [hunk 1]
```diff
@@ -9,97 +9,109 @@
...
-                    <img src="/Assets/Gryffindor-winter.png" class="small-card-img">
-                    <div class="small-card-content">
-                        <h4>New winter arrival</h4>
-                        <p>Check out the new winter wizard collection and upgrade your wadrobe!!!</p>
-                    </div>
+        </nav> 
+        <div class="main">
+            <div class="background dis-grid">
+                <div class="product centered product-gryffindor product-houses">
+                    <img src="/Assets/Gryffindor.png" class="house-image img-responsive">
```
Comment (raman1999) [inline index.html]: add alt attribute

**Review by raman1999 — APPROVED** [review]
everything else looks good to me

**Review by aishwarya-bm — APPROVED** [review]
Overall code looks clean & modular, just a suggestion to use utils wherever possible, like hide class for display : none and for positions

File: index.html (hunk 1 above)
Comment (nishasen) [reply index.html]: Yes will add that

**Comment by nishasen** [comment]
> Overall code looks clean & modular, just a suggestion to use utils wherever possible, like hide class for display : none and for positions

OKay I will change that

**Pull request merged**